cmd/ocsp-updater/main.go

```diff
@@ -268,6 +269,76 @@ func (updater *OCSPUpdater) oldOCSPResponsesTick(batchSize int) {
 	updater.generateOCSPResponses(statuses)
 }
 
+func (updater *OCSPUpdater) getSerialsIssuedSince(since time.Time, batchSize int) ([]string, error) {
+	var serials []string
+	_, err := updater.dbMap.Select(
+		&serials,
+		`SELECT serial FROM certificates
+		 WHERE issued > :since
+		 LIMIT :limit`,
+		map[string]interface{}{
+			"since": since,
+			"limit": batchSize,
+		},
+	)
+	if err == sql.ErrNoRows {
+		return serials, nil
+	}
+	return serials, err
+}
+
+func (updater *OCSPUpdater) getNumberOfReceipts(serial string) (int, error) {
+	var count int
+	err := updater.dbMap.SelectOne(
+		&count,
+		"SELECT COUNT(id) FROM sctReceipts WHERE certificateSerial = :serial",
+		map[string]interface{}{"serial": serial},
+	)
+	return count, err
+}
+
+// missingReceiptsTick looks for certificates without the correct number of SCT
+// receipts and retrieves them
+func (updater *OCSPUpdater) missingReceiptsTick(batchSize int) {
+	now := updater.clk.Now()
+	since := now.Add(-updater.oldestIssuedSCT)
+	serials, err := updater.getSerialsIssuedSince(since, batchSize)
+	if err != nil {
+		updater.log.AuditErr(err)
```

Comment:
one easy way to add context would be:

`AuditErr(fmt.Errorf("something went wrong: %s", err))`